src/qtiItem/helper/maxScore.js

```diff
@@ -121,11 +121,30 @@ export default {
                     maxScoreOutcome.buildIdentifier('MAXSCORE', false);
                 }
                 maxScoreOutcome.setDefaultValue(maxScore);
-            } else {
-                //remove MAXSCORE:
-                item.removeOutcome('MAXSCORE');
+
             }
-        }
+
+            //handle special case, when responseProcessing is set to none and MAXSCORE is setup manually
+            //remove MAXSCORE if externalProcessing is not set up:
+            maxScoreOutcome = item.getOutcomeDeclaration('MAXSCORE');
+            if(maxScoreOutcome) {
+                var interactions = item.responseProcessing.getRootElement().getInteractions();
+                if(interactions.length === 1) {
```

Comment:
why we need to check interactions and response?
I think we need only check for `externalScored`
```
if(maxScoreOutcome.attributes && maxScoreOutcome.attributes.externalScored) {
                            if(_.isUndefined(maxScoreOutcome.defaultValue)) {
                                maxScoreOutcome.setDefaultValue(1)
                            }
 } else {
                            item.removeOutcome('MAXSCORE');
                        }
```

Reply:
Because otherwise MAXSCORE will be removed when you setup response processing. Here I check that `responceProcessing` is set up to other value than `none`

Reply:
Just to remember the outcome of the alignment call: we should rely on `hasInvalidInteraction` while checking if we need to remove MAXSCORE  outcome or leave it